### PR TITLE
remove proto for non-proto-able resources

### PR DESCRIPTION
**Pull request by deads2k**
Proof for https://github.com/openshift/api/pull/102 .

/assign @sttts

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -38,7 +38,8 @@ import:
 
 # openshift second
 - package: github.com/openshift/api
-  version: master
+  repo:    https://github.com/deads2k/api.git
```
Comment (soltysh) [inline glide.yaml]: I hope you're not going to merge this as this.
/hold

Comment (deads2k) [reply glide.yaml]: > I hope you're not going to merge this as this.
> /hold

it won't merge with bad travis

**Pull request merged**